src/Console/Command/ObfuscateData.php

```diff
@@ -0,0 +1,225 @@
+            $this->info("This script will only run on test or staging deployments.\n\n");
+            exit;
+        }
+    }
+
+    private function isTestDeployment()
+    {
+        return (strtolower(getenv('APP_ENV')) == 'test');
+    }
+
+    private function isStagingDeployment()
+    {
+        return (strtolower(getenv('APP_ENV')) == 'staging');
+    }
+
+    /* optional implementations...
+    private function randomizeLongitudeLatitude($boundaries)
+    {
+
+    }
+    */
+
+    /* TODO: would this be useful?
+    private function scrambleLongitudeLatitude($distanceFromOrigin = null)
+    {
+
+    }
+    */
+
+    private function overwriteContacts()
+    {
+        $this->info("Overwriting contact data...");
+        $faker = Faker\Factory::create();
+        $query = DB::select('contacts.id', 'contacts.type', 'contacts.contact')
+            ->distinct(true)
+            ->from('contacts');
+        $results = $query->execute($this->db);
+        $overwrittenCount = 0;
+        $resultsCount = count($results);
+        foreach ($results as $row) {
+            $contactEntity = $this->contactRepository->getEntity($row);
+            $fakePhone = $faker->e164PhoneNumber;
+            $contactEntity->setState(['contact'=> $fakePhone]);
+            $overwrittenCount += $this->contactRepository->update($contactEntity);
+        }
+        if ($overwrittenCount == $resultsCount) {
+            $this->info("updated ".$overwrittenCount." records.\n");
+            return $overwrittenCount;
+        } else {
+            $this->info("failed to overwrite all records.\n");
+            return false;
+        }
+    }
+
+    private function overwritePostAuthors()
+    {
+        $this->info("Overwriting post author data...");
+        $faker = Faker\Factory::create();
+        $results = DB::select('posts.*')
+            ->from('posts')
+            ->execute($this->db)
+            ->as_array();
+        $overwrittenCount = 0;
+        $resultsCount = count($results);
+        foreach ($results as $row) {
+            $postEntity = $this->postRepository->getEntity($row);
+            $fakeEmail = $faker->safeEmail;
+            $fakeName = $faker->firstName." ".$faker->lastName;
+            $postEntity->setState([
+                'author_realname'=> $fakeName,
+                'author_email'=> $fakeEmail,
+            ]);
+            $overwrittenCount += $this->postRepository->update($postEntity);
+        }
+        if ($overwrittenCount == $resultsCount) {
+            $this->info("updated ".$overwrittenCount." records.\n");
+            return $overwrittenCount;
+        } else {
+            $this->info("failed to overwrite all records.\n");
+            return false;
+        }
+    }
+
+    private function overwriteSiteConfig()
+    {
+        $this->info("\nORemoving general settings data...");
+        $siteConfig = $this->configRepository->get('site');
+        $siteConfig->setState([
+            'name' => 'Test site',
```

Comment:
Not sure if we want to wipe the name and description. Maybe just prefix them with `SANITIZED` or similar. It'd be useful to still know what deployment you're looking at data from.